js/ui.js

```diff
@@ -1,242 +1,233 @@
-var $beginner_preset = true;
-var $intermediate_preset = false;
-var $advanced_preset = false;
+let $beginner_preset = true;
```

Comment:
(Same as above.) These shouldn't be declared with let as they are outside the block context and in the global context. These should remain var declared.